xml/bp_troubleshooting_tips.xml

```diff
@@ -782,7 +783,11 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
   <itemizedlist mark="bullet" spacing="normal">
    <listitem>
     <para>
-     NOTE: - &cephsalt; needs to implement network diagnostics.
+     Basic network diagnostics. Try diagnostics tools runner <literal>net.ping</literal>
+     to ping between cluster nodes to see if individual interface can reach to
+     specific interface and the average response time. Any specific response
+     time much slower then average will also be reported. See <xref linkend="network-test-basic-diagnostics"/>
+     for more information.
```

Comment:
hm, this seems to be missing the exact command the user would issue to "try diagnostics tools runner net.ping"?

Reply:
In any case, "try running net net.ping diagnostics tool. This tool has cluster nodes send network pings from their network interfaces to the network interfaces of the other cluster nodes, and measures the average response time." might be a better wording?